index.html

```diff
@@ -16,8 +16,39 @@
 </head>
 <body>
   <header>
+    <div class="container">
+      <div class="row">
+        <div class="column-full"></div>
+        <h3>Code Journal</h3>
+      </div>
+    </div>
+
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <form action="" method="get">
+        <h1>New Entry</h1>
+        <div class="row">
+          <img class="column-half" id="display-image" src="images/placeholder-image-square.jpg" alt="user-image">
+          <div class="column-half">
+            <div class="column-half">
```

Comment:
you don't need to set a column class inside of one, you just want to have the labels and inputs in here, and set them to be display block and 100% width so they take up the full column and stack